api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/GetConfiguredNodesOnEndpointResponseApiModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿// ------------------------------------------------------------
```

Comment:
I think we don't really need this model. For the response I think it would be enough to return the a list of PublishedNodeApiModel instances.